tests/conftest.py

```diff
@@ -111,6 +113,23 @@ def backend(request):
     await send({"type": "http.response.body", "body": body})
 
 
+async def echo_headers(scope, receive, send):
+    body: bytes = b""
+
+    for name, value in scope.get("headers", []):
+        value = f"{name.capitalize().decode()}: {value.decode()}\n"
+        body += value.encode()
```

Comment:
Nit: Maybe it'd be more ergonomic to return the headers as a JSON object? I think it would make the assertions more readable.

Reply:
Nice hint, thanks